external/external.cmake

```diff
@@ -72,6 +72,65 @@ include(minitrace)
 set(BUILD_EXAMPLES OFF CACHE BOOL "Disable Miniz examples" FORCE)
 add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/miniz)
 
+# Mono
+if (WIN32)
+	set(MONO_LIB_NAME libmono-static-sgen)
+else() 
+	set(MONO_LIB_NAME monosgen-2.0)
+endif()
+if (MONO_ROOT)
+	message("${MONO_ROOT}/lib")
+	message("${MONO_ROOT}/include/mono-2.0")
```

Comment:
Those debug message should probably be removed

Reply:
Yup, that escaped me.